src/components/CourseTemplateModal/EditTemplateForm.test.tsx

```diff
@@ -194,3 +225,45 @@ describe('EditTemplateForm Autofill Tests', () => {
     expect(tags).toHaveLength(0);
   });
 });
+
+describe("EditTemplateForm's Update Functinoality Tests", () => {
+  beforeEach(() => {
+    mockOnClose.mockClear();
+    (update as jest.Mock).mockClear();
+    displayContainer = renderWithTheme(
```

Comment:
In reference to earlier comment, displayContainer is not used later on in the code so I don't see the reason to restore the render in a variable